server/data/fr/gutenberg_33738.txt

```diff
@@ -0,0 +1,140 @@
+A Pépa, stances.
+SOUVENIRS DE Madame JAUBERT.
+Paris, Hetzel, mille huit cent quatre-vingt-un.
+Le Comte d'Essex, plan de tragédie.
+L'ÉVÉNEMENT, vingt et un novembre mille huit cent quatre-vingt-cinq.
+Alliance de la prose et de la poésie.
+LE VOLTAIRE, vingt-trois avril mille huit cent quatre-vingt-sept.
+(transcription) (acte quatre) (chez la reine) La reine, ses femmes.
+La Comtesse D'Essex vient implorer la grace de son mari.
+Pense un instant et pars joyeux, Ta maîtresse en sera plus belle.
+(Paris, mille huit cent soixante-quinze.
+In-8º.) Les Variantes de Venise, écrites pour être mises en musique par Gounod.
+Choudens, éditeur à Paris.
+Paris, Hachette, mille huit cent vingt-huit.
+_Revue Fantastique_,  deuxième article, premier février mille huit cent trente et un.
+GAUTIER (Charpentier, mille huit cent quatre-vingt-sept.
+in-8º, deux, p.
+L'ANGLAIS MANGEUR D'OPIUM, traduit de l'anglais par A.
+«Traduit» est certainement exagéré.
+cent trente-six), de la chambre (p.
+cent trente-neuf) et l'histoire des deux tasses de thé (p.
+Cet instant vint enfin : je croyais voir quelque chose de beaucoup plus horrible.
+La leçon commença : je riais de mes camarades que le mal de coeur prenait.
+Ces impressions, reçues dans ma jeunesse, donnèrent lieu à un rêve que j'avais assez fréquemment.
+«Il me semblait que j'étais couché et que je m'éveillais dans la nuit.
+Cependant, je n'en étais ni effrayé ni même étonné.
+Alors, je voyais entrer le même cadavre, que tout à l'heure j'avais trouvé par terre.
+Pourtant, il arrivait à moi sans parler et se couchait sur moi.
+Un autre rêve lui succédait.
+Je sentais ces poils dégoûtants m'effleurer le cou et le visage.
+Johny, braconnier                      Bouffé.
+Sténie, jeune fermier              Madame Albert.
+Miss Eveline, nièce du Laird.
+Gertrude, sa gouvernante               Florval.
+Écuyers, Piqueurs, Varlets., La scène est en Écosse.
+Devant une interdiction aussi impérieuse qu'inattendue, de la part de Madame H.
+Beau fiancé, je suis ta femme ; Emporte-moi dans ton mantel Jusqu'au foyer de ton chatel.
+Albert de Calvimont, musique de Gomis (Paris, Barba, mille huit cent trente-quatre.
+In-8º), dont le sujet est pris à la même source et l'intrigue presque identique.
+Combien Alfred de Musset en a-t-il écrit dans ce nombre ? Je l'ignore.
+Et cette collaboration anonyme ne s'est pas bornée au journal Le Temps.
+«quatorze mars mille huit cent trente-trois.
+Quelle sorcière ? «Au troisième acte, nous sommes en plein vent.
+mille huit cent trente et un.
+Chez Giraldon Bovinet, un vol.
+in-8º, qui fut mis en vente vers la fin de l'année mille huit cent trente.
+Voir : FÉLIX ARVERS, par Charles Glinel.
+Reims, OEuvres Posthumes, in-douze, p.
+deux cent quarante et un.
+MES HEURES PERDUES par Félix Arvers.
+Paris, Fournier, mille huit cent trente-trois.
+puis un autre…..
+on se battait toujours ; Et de faim ni de soif, nul ne sentait l'envie.
+Deux jours !…..
```

Comment:
strange ellipses

Reply:
Also comes from the original data: http://www.gutenberg.org/cache/epub/33738/pg33738.txt, with `.....`